libr/anal/p/anal_sbpf.c

```diff
@@ -0,0 +1,607 @@
+/* radare2 - LGPL - Copyright 2025 - Analysis plugin for Solana BPF */
+
+#include <r_anal.h>
+#include <r_lib.h>
+#include <r_bin.h>
+#include <r_util.h>
+#include <r_util/r_new_rbtree.h>
+
+#define SBPF_PROGRAM_ADDR 	0x100000000ULL
+#define SBPF_MAX_STRING_SIZE 0x100
+
+typedef struct {
+	ut64 addr;
+	ut64 xref_addr;
+} SbpfStringRef;
+
+// Global flag to track if analysis has been done
+static bool sbpf_strings_analyzed = false;
```

Comment:
do not use global vars. is this really necessary? imho this command should be executed by the user at will, not at plugin init time